client/my-sites/plans-grid/components/billing-timeframe.tsx

```diff
@@ -287,17 +289,43 @@ const DiscountPromotion = styled.div`
 	margin-top: 6px;
 `;
 
+interface RefundNoticeProps {
+	showRefundPeriod?: boolean;
+	planSlug: string;
+	billingPeriod: GridPlan[ 'pricing' ][ 'billingPeriod' ];
+}
+
+const RefundNotice = ( { planSlug, showRefundPeriod, billingPeriod }: RefundNoticeProps ) => {
+	const translate = useTranslate();
+
+	if ( ! showRefundPeriod || isFreePlan( planSlug ) ) {
+		return null;
+	}
+
+	return (
+		<>
+			<br />
+			{ translate( 'Refundable within %(dayCount)s days. No questions asked.', {
+				args: {
+					dayCount: billingPeriod === 365 ? 14 : 7,
+				},
+			} ) }
```

Comment:
@zaguiini / @Automattic/quake can you please help me confirm as of when we stopped showing the refund notice in the billing description? Is there any discussion/P2 I can follow on the decision to stop showing it under `/setup/new-hosted-site`? 🙇

Reply:
I'm surprised, as far as I know the refund notice should still be there for the hosting flow. I hadn't noticed it was gone!

The `showRefundPeriod` prop is [still being set here for our flow](https://github.com/Automattic/wp-calypso/blob/bf76b613c6419a3114a38bdd6dc9a29b247e7419/client/my-sites/plans-features-main/index.tsx#L854).

[But I see `<FeaturesGrid>` isn't anything with this prop](https://github.com/Automattic/wp-calypso/blob/bf76b613c6419a3114a38bdd6dc9a29b247e7419/packages/plans-grid-next/src/components/features-grid/index.tsx#L219). Is this a regression introduced during the refactor to `@automattic/plans-grid-next`?

Reply:
Interesting. It sounds like a regression then. I'll create an issue and perhaps @southp can coordinate with fixing? Otherwise, I'll get to it when back.

Thanks for confirming @p-jackson 🙇

Reply:
Created issue here https://github.com/Automattic/wp-calypso/issues/89945

@southp can you coordinate with investigating? It's likely that we broke this in the refactors.

Reply:
https://github.com/Automattic/wp-calypso/pull/90056